pkg/config/config.go

```diff
@@ -158,11 +159,18 @@ func (c *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	// We want to set c to the defaults and then overwrite it with the input.
 	// To make unmarshal fill the plain data struct rather than calling UnmarshalYAML
 	// again, we have to hide it using a type indirection.
+
+	// TODO: Handle the empty defaults case
```

Comment:
What do you mean?

Reply:
If there is no `defaults` section, jiralert panics

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x13c40b5]

goroutine 1 [running]:
gopkg.in/yaml%2ev2.handleErr(0xc000157d58)
	/Users/rporresm/go/1.15.15/pkg/mod/gopkg.in/yaml.v2@v2.3.0/yaml.go:249 +0x85
panic(0x15b0f60, 0x1a250c0)
	/Users/rporresm/.goenv/versions/1.15.15/src/runtime/panic.go:969 +0x1b9
github.com/prometheus-community/jiralert/pkg/config.(*Config).UnmarshalYAML(0xc00010f500, 0xc00000d860, 0x0, 0x94b6b18)
	/Users/rporresm/git/github.com/rporres/jiralert/pkg/config/config.go:170 +0x75
gopkg.in/yaml%2ev2.(*decoder).callUnmarshaler(0xc00002f080, 0xc000175ea0, 0x94b6b18, 0xc00010f500, 0xc00010f500)
	/Users/rporresm/go/1.15.15/pkg/mod/gopkg.in/yaml.v2@v2.3.0/decode.go:270 +0xa6
gopkg.in/yaml%2ev2.(*decoder).prepare(0xc00002f080, 0xc000175ea0, 0x1606360, 0xc00010f500, 0x199, 0x104e0a5, 0xc00000e140, 0x0, 0x0)
	/Users/rporresm/go/1.15.15/pkg/mod/gopkg.in/yaml.v2@v2.3.0/decode.go:313 +0x1d0
gopkg.in/yaml%2ev2.(*decoder).unmarshal(0xc00002f080, 0xc000175ea0, 0x1606360, 0xc00010f500, 0x199, 0x0)
	/Users/rporresm/go/1.15.15/pkg/mod/gopkg.in/yaml.v2@v2.3.0/decode.go:364 +0xe9
gopkg.in/yaml%2ev2.(*decoder).document(0xc00002f080, 0xc000175e30, 0x1606360, 0xc00010f500, 0x199, 0x13913a7)
	/Users/rporresm/go/1.15.15/pkg/mod/gopkg.in/yaml.v2@v2.3.0/decode.go:384 +0x7c
gopkg.in/yaml%2ev2.(*decoder).unmarshal(0xc00002f080, 0xc000175e30, 0x1606360, 0xc00010f500, 0x199, 0x199)
	/Users/rporresm/go/1.15.15/pkg/mod/gopkg.in/yaml.v2@v2.3.0/decode.go:360 +0x250
gopkg.in/yaml%2ev2.unmarshal(0xc000132800, 0x3f4, 0x400, 0x15c02c0, 0xc00010f500, 0x400, 0x0, 0x0)
	/Users/rporresm/go/1.15.15/pkg/mod/gopkg.in/yaml.v2@v2.3.0/yaml.go:148 +0x33d
gopkg.in/yaml%2ev2.Unmarshal(...)
	/Users/rporresm/go/1.15.15/pkg/mod/gopkg.in/yaml.v2@v2.3.0/yaml.go:81
github.com/prometheus-community/jiralert/pkg/config.Load(0xc000132400, 0x3f4, 0x3f4, 0xc000132400, 0x3f4)
	/Users/rporresm/git/github.com/rporres/jiralert/pkg/config/config.go:53 +0x99
github.com/prometheus-community/jiralert/pkg/config.LoadFile(0x1657b30, 0x13, 0x16fb140, 0xc000286e10, 0x0, 0x0, 0xc000286f00, 0x0, 0x0, 0x0)
	/Users/rporresm/git/github.com/rporres/jiralert/pkg/config/config.go:67 +0x214
main.main()
	/Users/rporresm/git/github.com/rporres/jiralert/cmd/jiralert/main.go:74 +0x2a5

Reply:
Clarified comment in b7f35ac